### PR TITLE
first changes

### PR DESCRIPTION
**Pull request by LazarevSlava**
# Added

- function for moving map 

## Checklist

- [x] Linter checks
- [x] Code style checks

### CODE REVIEW

**Comment by LazarevSlava** [comment]
Please help me a little
Check my comment in the file MyComponent and tell me please where is my fault?
How to use only useMapEvent without other functions?

File: src/components/leaflet/LeafletMap.jsx [hunk 1]
```diff
@@ -2,6 +2,7 @@ import { MapContainer, TileLayer, Marker } from 'react-leaflet';
 import style from './leaflet.module.scss';
 import { Icon } from 'leaflet';
 import MarkerIcon from '../../assets/images/icon-location.svg';
+import { MyComponent } from '../../helpers/myComponent';
```
Comment (michey85) [inline src/components/leaflet/LeafletMap.jsx:5]: Is there a better name for the component?

File: src/components/leaflet/LeafletMap.jsx [hunk 2]
```diff
@@ -13,12 +14,13 @@ const customIcon = new Icon({
 const LeafletMap = (props) => {
   return (
     <MapContainer
-      key={`${props.coordinates[0]}`}
+      // key={`${props.coordinates[0]}`}
```
Comment (michey85) [inline src/components/leaflet/LeafletMap.jsx:17]: It has to be removed not just commented out.

File: src/helpers/myComponent.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+import { useMap } from 'react-leaflet';
+
+export function MyComponent({ geoInfo }) {
+  const map = useMap();
+
+  useEffect(() => {
+    if (geoInfo) {
+      map.setView(geoInfo, map.getZoom());
+    }
+  }, [geoInfo, map]);
```
Comment (michey85) [inline src/helpers/myComponent.js:31]: What doesn't work for you here? As I can see you haven't said anything about animation here.
If you look at `mat.setView` function declaration you see the third parameter which is an object with 4 possible properties:
```
    animate?: boolean | undefined;
    duration?: number | undefined;
    easeLinearity?: number | undefined;
    noMoveStart?: boolean | undefined;
```

File: src/components/leaflet/LeafletMap.jsx (hunk 2 above)
Comment (LazarevSlava) [reply src/components/leaflet/LeafletMap.jsx]: done

File: src/components/leaflet/LeafletMap.jsx (hunk 1 above)
Comment (LazarevSlava) [reply src/components/leaflet/LeafletMap.jsx]: done

File: src/helpers/myComponent.js (hunk 3 above)
Comment (LazarevSlava) [reply src/helpers/myComponent.js]: I used map.flyTo

**Review by michey85 — APPROVED** [review]

**Pull request merged**